ios/RNDocumentPicker/RNDocumentPicker.m

```diff
@@ -54,25 +59,71 @@ - (dispatch_queue_t)methodQueue
                   rejecter:(RCTPromiseRejectBlock)reject)
 {
     NSArray *allowedUTIs = [RCTConvert NSArray:options[OPTION_TYPE]];
-    UIDocumentPickerViewController *documentPicker = [[UIDocumentPickerViewController alloc] initWithDocumentTypes:(NSArray *)allowedUTIs inMode:UIDocumentPickerModeImport];
+    
+    UIDocumentMenuViewController *documentPicker = [[UIDocumentMenuViewController alloc] initWithDocumentTypes:(NSArray *)allowedUTIs inMode:UIDocumentPickerModeImport];
```

Comment:
And you've reverted the fix to this library's behaviour to follow Apple's recommended method of opening the picker, and restored the old behaviour that Apple has explicitly deprecated.

Reply:
@adrianicv, could you be able to revert this changes?